master/internal/rm/kubernetesrm/informer.go

```diff
@@ -81,28 +47,26 @@ func (i *informer) startInformer(ctx *actor.Context) {
 		},
 	})
 	if err != nil {
-		ctx.Log().WithError(err).Warnf("error initializing pod retry watcher")
-		actors.NotifyAfter(ctx, defaultInformerBackoff, startInformer{})
-		return
+		i.syslog.WithError(err).Warnf("error initializing pod retry watcher")
 	}
 
-	ctx.Log().Info("pod informer is starting")
+	i.syslog.Info("pod informer is starting")
 	for event := range rw.ResultChan() {
 		if event.Type == watch.Error {
-			ctx.Log().Warnf("pod informer emitted error %+v", event)
+			i.syslog.Warnf("pod informer emitted error %+v", event)
 			continue
 		}
 
 		pod, ok := event.Object.(*k8sV1.Pod)
 		if !ok {
-			ctx.Log().Warnf("error converting event of type %T to *k8sV1.Pod: %+v", event, event)
+			i.syslog.Warnf("error converting event of type %T to *k8sV1.Pod: %+v", event, event)
 			continue
 		}
 
-		ctx.Log().Debugf("informer got new pod event for pod: %s %s", pod.Name, pod.Status.Phase)
-		ctx.Tell(i.podsHandler, podStatusUpdate{updatedPod: pod})
+		i.syslog.Debugf("informer got new pod event for pod: %s %s", pod.Name, pod.Status.Phase)
+		return pod, true
```

Comment:
this returns after the first event, but the informer may emit many events from a single `rw.ResultChan`. we will need to use a callback or something, here, to fire the old `ctx.Tell(i.podsHandler, podStatusUpdate{updatedPod: pod})`